includes/admin/views/review.php

```diff
@@ -0,0 +1,94 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'You are not allowed to call this page directly.' );
+}
+?>
+<div class="notice notice-info is-dismissible wpbdp-review-notice">
+	<div class="wpbdp-satisfied">
+		<p>
+			<?php echo esc_html( $title ); ?>
+			<br/>
+			<?php esc_html_e( 'Are you enjoying Business Directory Plugin?', 'business-directory-plugin' ); ?>
+		</p>
+		<a href="#" class="wpbdp_reverse_button wpbdp_animate_bg show-wpbdp-feedback wpbdp-button-secondary" data-link="feedback">
+			<?php esc_html_e( 'Not Really', 'business-directory-plugin' ); ?>
+		</a>
+		<a href="#" class="wpbdp_animate_bg show-wpbdp-feedback wpbdp-button-primary" data-link="review">
+			<?php esc_html_e( 'Yes!', 'business-directory-plugin' ); ?>
+		</a>
+	</div>
+	<div class="wpbdp-review-request hidden">
+		<p><?php esc_html_e( 'Awesome! Could you do me a BIG favor and give Business Directory Plugin a review to help me grow my little business and boost our motivation?', 'business-directory-plugin' ); ?></p>
+		<p>- Steph Wells<br/>
+			<span><?php esc_html_e( 'Co-Founder and CTO of Business Directory Plugin', 'business-directory-plugin' ); ?><span>
+		</p>
+		<a href="#" class="wpbdp-dismiss-review-notice wpbdp_reverse_button wpbdp-button-secondary" data-link="no" target="_blank" rel="noopener noreferrer">
+			<?php esc_html_e( 'No thanks, maybe later', 'business-directory-plugin' ); ?>
+		</a>
+		<a href="https://wordpress.org/support/plugin/business-directory-plugin/reviews/?filter=5#new-post" class="wpbdp-dismiss-review-notice wpbdp-review-out wpbdp-button-primary" data-link="yes" target="_blank" rel="noopener">
+			<?php esc_html_e( 'Ok, you deserve it', 'business-directory-plugin' ); ?>
+		</a>
+		<br/>
+		<a href="#" class="wpbdp-dismiss-review-notice" data-link="done" target="_blank" rel="noopener noreferrer">
+			<?php esc_html_e( 'I already did', 'business-directory-plugin' ); ?>
+		</a>
+	</div>
+	<div class="wpbdp-feedback-request hidden">
+		<p><?php esc_html_e( 'Sorry to hear you aren\'t enjoying building with Business Directory Plugin. We would love a chance to improve. Could you take a minute and let us know what we can do better?', 'business-directory-plugin' ); ?></p>
+
+		<div id="wpbdpapi-feedback" class="wpbdpapi-form" data-url="https://services.strategy11.com/wp-json/frm/v2/forms/bd-feedback?return=html&exclude_script=jquery">
+			<span class="wpbdp-wait wpbdp_visible_spinner"></span>
+		</div>
+	</div>
+</div>
+<script type="text/javascript">
+	jQuery( document ).ready( function( $ ) {
+		$(document).on( 'click', '.wpbdp-dismiss-review-notice, .wpbdp-review-notice .notice-dismiss', function( event ) {
```

Comment:
Many space characters missing in the JS code. Please follow WordPress JS coding standard: https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/

Variables should be declared at the top of each function.

Reply:
The JS in this case cannot have the variables declared at the top as they are used for DOM checks and used once where applicable

Reply:
I think to prevent js variable conflicts, in this case it would be better not to have them global as we do not know what else will be called or will use the same variable name. 
The modal is loaded in the admin notification, so we would need to define custom js variables, which at this case I do not think are necessary. It would be best to keep them as local variables in this instance